google-cloud-pubsub/CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Release History
 
+### 1.1.0 / 2019-10-23
+
+#### Features
+
+* Add support for Ordering Keys
+  * Google Cloud Pub/Sub ordering keys provide the ability to ensure related
+    messages are sent to subscribers in the order in which they were published.
+    The service guarantees that, for a given ordering key and publisher, messages
+    are sent to subscribers in the order in which they were published.
+  * Note: At the time of this release, ordering keys are not yet publicly enabled
```

Comment:
ordering keys is not

Reply:
Really? This wording is used several times in the documentation, not just in this CHANGELOG entry.